airflow/utils/db.py

```diff
@@ -1169,7 +1171,7 @@ def upgradedb(
 
         previous_revision = _get_current_revision(session=session)
 
-        log.info("Creating tables")
+        log.info("Upgrading the Airflow database")
```

Comment:
```suggestion
        log.info("Migrating the Airflow database")
```

Should we use migrating instead of upgrading?